src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -313,9 +370,14 @@ export const TradeInput = memo(() => {
                 amountBeforeFeesCryptoPrecision={buyAmountBeforeFeesCryptoPrecision}
                 protocolFees={totalProtocolFees}
                 shapeShiftFee={shapeShiftFee}
-                donationAmount={donationAmount}
+                donationAmountUserCurrency={
+                  shapeShiftFee?.amountAfterDiscountUserCurrency ??
+                  donationAmountUserCurrency ??
+                  '0'
+                }
                 slippageDecimalPercentage={slippageDecimal}
                 swapperName={activeSwapperName ?? ''}
+                defaultIsOpen={true}
```

Comment:
```suggestion
                defaultIsOpen
```

Reply:
Would prefer to keep it `true` for declarativeness 🙏🏽